test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -261,10 +265,14 @@ private static void validateSpec(RestSpec restSpec) {
 
     @After
     public void wipeCluster() throws Exception {
-
         // wipe indices
+        String deleteIndices = "*";
+        if (excludeIndices().size() > 0) {
+            deleteIndices = Strings.collectionToDelimitedString(excludeIndices(), ",", "-", "");
```

Comment:
nit: how about being explicit and prepending the list of exclusions with `*`? e.g. `*,-index,-index2`